LoggingKata/TacoParser.cs

```diff
@@ -9,10 +9,32 @@ public class TacoParser
         
         public ITrackable Parse(string line)
         {
-            logger.LogInfo("Begin parsing");
+            //logger.LogInfo("Begin parsing");
+            TacoBell tacoBell = new TacoBell();
+            Point point = new Point();
```

Comment:
These instantiations might be better, organizationally, if they were down closer to where they're used around line 24